api/Dockerfile

```
@@ -15,6 +13,7 @@ RUN \
     pip install -r requirements.txt && \
     apk --purge del .build-deps
 
-COPY ./lokajapi .
+WORKDIR /lokajapi
+COPY ./lokajapi/ /lokajapi
```

tu poprawiłeś mój błąd, można by też zrobić komendę `ADD` która dodaje zawartość katalogu do zadanego katalogu

```suggestion
ADD ./lokajapi .
```

---

Okke.